test/examples/ucp_client_server.c

```diff
@@ -405,6 +446,8 @@ static int start_server(ucp_worker_h ucp_worker, ucx_server_ctx_t *context,
     ucp_listener_params_t params;
     ucp_listener_attr_t attr;
     ucs_status_t status;
+    char ip_str[50];
```

Comment:
@shamisp Inside ucx, we use this #define - https://github.com/openucx/ucx/blob/master/src/ucs/sys/sock.h#L24
but since this is an example code and both usages are in the same function and only for a print, IMHO it's not really necessary.